dataloader/tasks_segments_parallels.py

```diff
@@ -104,12 +105,13 @@ def get_folios_from_segment_keys(segment_keys, lang):
     elif lang == LANG_SANSKRIT:
         last_num = ''
         for segment_key in segment_keys:
-            if re.search(r"^(XXdhppat|S10udanav)", segment_key):
-                num = segment_key.split(":")[1].split("_")[1]
-                if num != last_num:
-                    folios.append({"num": num, "segment_nr": segment_key})
-                    last_num = num
-            elif re.search(r"^(OT)", segment_key):
+            # TODO: Ask Vimala whether we need this?
```

Comment:
Yes, we do. These files have different segment numbers that need to be parsed a bit differently.

Reply:
I've changed this already.

Reply:
Great!